packages/react-magma-dom/src/components/Table/Table.tsx

```diff
@@ -122,6 +127,24 @@ export const TableContainer = styled.div<{
           : props.theme.colors.focus};
   }
 `;
+export const StyledCaption = styled.caption<{ isInverse: boolean }>`
+  color: ${props =>
```

Comment:
If we are copying basically all the styles from `headingMediumStyles`, we should be able to do something like this so we don't duplicate code and if one gets updated, they both change simultaneously:
```
export const StyledCaption = styled.caption<{ isInverse: boolean }>`
    ${headingMediumStyles(props)}
    display: flex;
    flex: 1;
`;
```